src/ctype/CType.utils.ts

```diff
@@ -34,7 +34,10 @@ export function verifySchemaWithErrors(
   return !!result
 }
 
-export function verifySchema(object: object, schema: object): boolean {
+export function verifySchema(
+  object: Record<string, any>,
```

Comment:
Looks like a use case for `unknown`?

```suggestion
  object: Record<string, unknown>,
```

Reply:
[See above](https://github.com/KILTprotocol/sdk-js/pull/273#discussion_r444200331).